odl/solvers/util/callback.py

```diff
@@ -315,6 +315,9 @@ def __init__(self, *args, **kwargs):
         ----------
         display_step : positive int, optional
             Number of iterations between plots. Default: 1
+        saveto : string, optional
```

Comment:
Also should be a path, not a string. See https://docs.python.org/2/library/os.path.html